sdk/storage/azure-storage-file-datalake/src/main/java/com/azure/storage/file/datalake/models/ListPathsOptions.java

```diff
@@ -53,10 +53,10 @@ public boolean isRecursive() {
      * The default value is false. Note that group and application Object IDs are not translated because they do not
      * have unique friendly names.
      *
-     * @return the returnUpn value.
+     * @return the returnUserPrincipalName value.
      */
-    public boolean isReturnUpn() {
-        return returnUpn;
+    public boolean returnsUserPrincipalName() {
```

Comment:
I'm leaning towards just calling this `returnUserPrincipalName` as that already has a boolean-esque connotation.